### DISCUSSION TITLE
Use MIKE1D SDK with MIKE+

### DISCUSSION THREAD

**Discussion by hawklorry** [DHI/Mike1D-SDK-examples#15] [Q&A] [answered]
Hi, I'm working on a project with MIKE+. One task is to calibrate the model with six events. As MIKE+ doesn't allow multiple simulations, I'm trying to run it with MIKE1D SDK. The example and document is good but doesn't mention the format of simulation file. If I have the scenario name I want to run, how could I use MIKE1D SDK to run the simulations? Do I need to create the simulation files? I'd appreciate your help.

**Comment by JesperGr** [comment, marked as answer]
With the MIKE 1D engine it is possible to specify which MIKE+ similation to run, in case more than one simulation is defined in the database, by specifying the ID of the simulation. 

It is not possible using the MIKE 1D engine to change the scenario within MIKE+. So, if your events are defined as scenarios, these must be activated from within MIKE+ user interface. When activated, then it is possible to export  the database to simulation files and run the MIKE 1D engine from there.

**Comment by hawklorry** [comment]
Thanks Jesper. By export, do you mean export to m1dx file?

**Comment by JesperGr** [comment]
Yes, I mean export to m1dx file.
